src/gui/components/config/GraphComponent.h

```diff
@@ -598,14 +669,17 @@ bool GraphComponent<ValueType>::keyPressed(
 {
     if (key.isKeyCode(juce::KeyPress::backspaceKey))
     {
-        pointEditState = (pointEditState == PointEditingState::Delete)
-                             ? PointEditingState::None
-                             : PointEditingState::Delete;
-
+        pointEditState = PointEditingState::Delete;
```

Comment:
This contains some code from the other PR - can they be separated? Ideally this should be independent of your other PR :)